src/stp/evaluations/position/BlockingEvaluation.cpp

```diff
@@ -12,7 +12,7 @@ uint8_t BlockingEvaluation::metricCheck(double pointDistance, std::vector<double
     double maxAngle = M_PI / 6;  // 30 degrees, max angle considered for blocking
```

Comment:
Please make this constexpr so this will be calculated during compilation (in DEBUG mode. Release mode will most likely already optimize it)